src/libumf.map

```diff
@@ -117,3 +114,9 @@ UMF_1.0 {
     local:
         *;
 };
+
+UMF_0.11 {
+        umfFixedMemoryProviderOps;
```

Comment:
do we have any other new symbols on main...?

Reply:
I'm not what about recent change in umf_memory_provider_ops_t struct and functions using it  but we'll figure it out in backward comp. tests

Reply:
heh, I guess we have to figure it out, cause it's already failing on my test branch in UR 😄 

https://github.com/oneapi-src/unified-runtime/actions/runs/12710368809/job/35431309282?pr=2546

Reply:
> heh, I guess we have to figure it out, cause it's already failing on my test branch in UR 😄
> 
> https://github.com/oneapi-src/unified-runtime/actions/runs/12710368809/job/35431309282?pr=2546

@lukaszstolarczuk Is it still failing ?